adding in suggested tags feature beta banner as well as new demo video (#15103)
DataDog/documentation: content/en/metrics/metrics-without-limits.md

Click on any metric name to open its details sidepanel. Then click **Manage Tags** -> **“Custom”** to configure the tags you’d like to remain as queryable on dashboards and monitors. Before selecting **Save**, an estimated new volume of indexed custom metrics that results from this potential tag configuration is displayed.

{{< img src="metrics/mwl_tag_config.mp4" alt="Configuration of Tags" video=true >}}

> As a beta feature of Metrics without Limits, we can prepopulate the tag configuration modal by default with an allowlist of tags that are actively queried on dashboards/notebooks/monitors for the past 30 days (colored in blue with an icon). You may include additional tags as well. 

Reviewer: ```suggestion
As a beta feature of Metrics without Limits, you can pre-populate the tag configuration modal by default with an allowlist of tags that have been actively queried on dashboards, notebooks, or monitors in the past 30 days (colored in blue with an icon). You may include additional tags as well. 
```

You can add or remove aggregations at any time with no required Agent or code-level changes. 

> As a beta feature of Metrics without Limits, we can prepopulate the tag configuration modal with an allowlist of aggregations that are actively queried on dashboards/notebooks/monitors for the past 30 days (colored in blue with an icon). 

Reviewer: ```suggestion
As a beta feature of Metrics without Limits, you can pre-populate the tag configuration modal with an allowlist of aggregations that have been actively queried on dashboards, notebooks, or monitors in the past 30 days (colored in blue with an icon). 
```
